bokehjs/src/lib/models/formatters/log_tick_formatter.ts

```diff
@@ -74,10 +82,15 @@ export class LogTickFormatter extends TickFormatter {
 
     const base = this.ticker?.base ?? 10
     const expos = this._exponents(ticks, base)
-
     if (expos == null)
       return this.basic_formatter.doFormat(ticks, opts)
     else
-      return expos.map((expo) => unicode_replace(`${base}^${expo}`))
+      return expos.map((expo) => {
+        if (abs(expo)<this.min_exponent)
```

Comment:
Jut thinking out loud, if the default `min_exponent` is 0, perhaps this should be `<=` comparison so that "1" just renders as "1" ? 

cc @bokeh/dev also what should the default be? (what is MPLs default?)

Reply:
In matplotlib the default is zero as well.

Reply:
Sounds good to me, I think the only question is `<` vs `<=` then